x-pack/legacy/plugins/canvas/public/components/popover/popover.tsx

```diff
@@ -9,7 +9,26 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { EuiPopover, EuiToolTip } from '@elastic/eui';
 
-export class Popover extends Component {
+interface Props {
```

Comment:
Let me know when you'd like me to test/review these changes.  I assume once you remove it from Draft status, but offering earlier, if you like...!

Reply:
Changes in this file and addition of `React.MouseEventHandler` was because otherwise TypeScript complains.